v2/jsonpatch.go

```diff
@@ -64,6 +64,9 @@ func NewOperation(op, path string, value interface{}) Operation {
 //
 // An error will be returned if any of the two documents are invalid.
 func CreatePatch(a, b []byte) ([]Operation, error) {
+	if reflect.DeepEqual(a, b) {
```

Comment:
May be `bytes.Equal`? 

https://pkg.go.dev/bytes#Equal

Reply:
Thank you! Yes, `bytes.Equal`. Updated.